WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
```

Comment:
> Medieval society (including in the Dark Ages) was not some perpetually monolithic epoch of cruelty and oppression.

Correct (although it always pains me when you say "Dark Ages").

> Kind rulers did exist, and that should be reflected in that Decision.

Kindness is not represented through religious tolerance towards the Jewish faith.